lib/task-data/tasks/catalog-smart.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
         commands: [
             {
                 command: 'sudo bash get_smart.sh',
-                downloadUrl: '/api/current/templates/get_smart.sh'
+                downloadUrl: '{{api.templates}}/get_smart.sh?nodeId={{context.target}}'
```

Comment:
This suggestion has gone beyond this PR, but ```{{ context.target }}``` gives people unclear hint that is actually referring to a node ID. something like ```?nodeId={{context.nodeId}}'``` or ```?nodeId={{ tasks.nodeID}}'``` would make the code more readable and less error-prong. 

Agreed? @yyscamper @anhou . If yes, this would generate a follow up refactor task.